CHI_TECH/ChiMesh/Cell/cell.h

```diff
@@ -4,6 +4,26 @@
 #include"../chi_mesh.h"
 #include <tuple>
 
+//Appending cell types to namespace
+namespace chi_mesh
+{
+  enum class CellTypes
```

Comment:
Since we now need to prefix each cell type with the enum class name, it would make more sense to remove the *_CELL* from the cell names, and maybe also change the enum name to *CellType*. IMHO this would read better:

```c++
CellType cell = CellType::TRIANGLE;
```

than

```c++
CellTypes cell = CellTypes::TRIANGLE_CELL;
```